kani-driver/src/call_cbmc_viewer.rs

```diff
@@ -70,6 +70,15 @@ impl KaniSession {
         // Let the user know
         if !self.args.quiet {
             println!("Report written to: {}/html/index.html", report_dir.to_string_lossy());
+            // If using VS Code with Remote-SSH, suggest an option for remote viewing:
+            if std::env::var("VSCODE_IPC_HOOK_CLI").is_ok()
+                && std::env::var("SSH_CONNECTION").is_ok()
+            {
+                println!(
+                    "VSCode Remote-SSH port forwards for you. Try:  python3 -m http.server --directory {}/html",
```

Comment:
Can you please improve this message? It's not clear to me what the message is here.

I'm assuming you are trying to say that there is a VSCode remote-ssh session running and the user might be able to see the trace by running the command above.

Also, do we have a more detailed documentation that we can provide a link to?

Reply:
That's better, thanks!